kibble/models/site_brand_test.go

```diff
@@ -0,0 +1,158 @@
+//    Copyright 2018 SHIFT72
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetImageOnEmpty(t *testing.T) {
```

Comment:
Could probably improve the test name a bit.  Something like `TestGetImageWhenSiteBrandImagesIsEmpty`.  Same for the others below.